cmd/paratime/show.go

```diff
@@ -35,15 +41,25 @@ const (
 const (
 	selInvalid propertySelector = iota
 	selParameters
+	selEvents
 )
 
+var eventDecoders = []func(*types.Event) ([]client.DecodedEvent, error){
+	accounts.DecodeEvent,
+	consensusaccounts.DecodeEvent,
+	contracts.DecodeEvent,
+	core.DecodeEvent,
+	evm.DecodeEvent,
+	rofl.DecodeEvent,
+}
+
 var (
 	outputFormat  string
 	selectedRound uint64
 
 	showCmd = &cobra.Command{
-		Use:     "show { <round> [ <tx-index> | <tx-hash> ] | parameters }",
-		Short:   "Show information about a ParaTime block, its transactions or other parameters",
+		Use:     "show { <round> [ <tx-index> | <tx-hash> ] | parameters | events }",
```

Comment:
These commands are currently inconsistent regarding the `round` argument.  `paratime show parameters` uses `--round` flag, while `paratme show <round>` uses `round` as a positional argument.

`show events` is consistent with `show parameters` since that was the easiest to implement. But this should probably be cleaned up (in a followup).

Reply:
Initially it was not meant to show other things, so if we now have parameters and events, we should maybe also have `show block` or similar.

Reply:
The initial idea was to resemble `oasis network show`. So try to autodetect what user wanted with as little typing as possible and then offer more explicit variants, if needed.

Some alternative ideas:
- `show <round> --events`
- `show events:<round>`
- `show <round>:events`
- or simply show all transactions with events when calling `show <round>`, also pass `--format json` and let user filter them with `jq`

IMO the current implementation makes the most sense.

Reply:
I'll merge it as is for now.